src/python/tests/database/test_dbconnection_lite.py

```diff
@@ -0,0 +1,116 @@
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Unit testing of `ensembl.io.genomio.database.dbconnection_lite` module.
+"""
+
+from pathlib import Path
+from typing import Optional
+from unittest.mock import patch
+import pytest
+
+from sqlalchemy import create_engine
+from sqlalchemy.engine import URL
+from sqlalchemy.orm import Session
+
+from ensembl.io.genomio.database import DBConnectionLite
+from ensembl.core.models import Base, Meta
+
```

Comment:
```suggestion


```
I may be wrong, but it should be 2 spaces between imports and constants based on PEP8.

Reply:
2 empty lines are only around top level classes and functions, cf https://peps.python.org/pep-0008/#blank-lines